src/sysupdate/sysupdate-util.h

```diff
@@ -3,3 +3,7 @@
 #pragma once
 
 int reboot_now(void);
+
+typedef enum SysupdateFlags {
+        SYSUPDATE_OFFLINE = 1 <<  0,
+} SysupdateFlags;
```

Comment:
This was cribbed from `PortableFlags` in `src/portable/portable.h`, which I was (via its man page) pointed at for this.  Since it's unusual though, I'll change to using a `#define`.